src/styles/utils/variables.scss

```diff
@@ -0,0 +1,12 @@
+$product-main-color: #060b35;
+$product-seconday-color: #616070;
```

Comment:
There is a typo in the variable name `$product-seconday-color`. It should be `$product-secondary-color` to maintain consistency and avoid potential errors.